client/js/mini-app/questions/resource-carousel.js

```diff
@@ -25,10 +25,9 @@ const ResourceCarousel = ({ classes, onResourceChange }) => (
 					<button
 						className={classes.resourceBox}
 						onClick={() => onResourceChange(resources[0].id)}
-						disabled
 					>
 						<div className={classes.flexColumn}>
-							<div className={classes.disabledIcon}>
+							<div className={classes.chcIcon}>
```

Comment:
@vzajkov we're keeping this disabled until we're able to verify all of the community health centers. Sorry for the confusion!